src/core/properties/cameraproperty.cpp

```diff
@@ -355,6 +384,37 @@ void CameraProperty::invokeEvent(Event* event) {
     }
 }
 
+inline Property* CameraProperty::getCameraProperty(const std::string& identifier) const {
+    const auto it =
+        std::find_if(cameraProperties_.begin(), cameraProperties_.end(),
+                     [&](auto property) { return property->getIdentifier() == identifier; });
+    if (it != cameraProperties_.end()) {
+        return *it;
+    } else {
+        return nullptr;
+    }
+}
+
+inline void CameraProperty::addCamerapProperty(Property* camprop) {
+    cameraProperties_.push_back(camprop);
+    ownedCameraProperties_.emplace_back(camprop);
+    insertProperty(size() - 1, camprop, false);
+}
+
+void CameraProperty::serialize(Serializer& s) const {
+    CompositeProperty::serialize(s);
+    s.serialize("Camera", camera_);
+
+}
+
+void CameraProperty::deserialize(Deserializer& d) {
+    hideConfiguredProperties();
```

Comment:
Why this hiding? Might want to clarify it’s purpose.

Reply:
Mmm will add some comment